<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>微表情识别</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="faceGroupBox">
        <property name="title">
         <string>人脸识别分类器：</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="faceEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="expressionGroupBox">
        <property name="title">
         <string>表情标签（表情之间用逗号隔开）：</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLineEdit" name="expressionEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gaborGroupBox">
        <property name="title">
         <string>Gabor参数：</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="kernelLabel">
             <property name="text">
              <string>核函数大小：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="kernelEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="varianceLabel">
             <property name="text">
              <string>方差：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="varianceEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="phaseLabel">
             <property name="text">
              <string>相位：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="phaseEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="directionLabel">
             <property name="text">
              <string>方向：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="directionEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="scaleLabel">
             <property name="text">
              <string>尺度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="scaleEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="paraGroupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="pcaLabel">
             <property name="text">
              <string>PCA维度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="pcaEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="ldaLabel">
             <property name="text">
              <string>LDA维度：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ldaEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="svmLabel">
             <property name="text">
              <string>SVM参数：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="svmEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="imageGroupBox">
        <property name="title">
         <string>图片归一化大小：</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="xLabel">
             <property name="text">
              <string>x:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="xEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="yLabel">
             <property name="text">
              <string>y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="yEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="buttonsGroupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="loadTrainButton">
             <property name="text">
              <string>载入训练图片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loadTestButton">
             <property name="text">
              <string>载入测试图片</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="trainButton">
             <property name="text">
              <string>训练</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="testButton">
             <property name="text">
              <string>测试</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="videoButton">
             <property name="text">
              <string>视频</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="gaborTestButton">
             <property name="text">
              <string>Gabor测试</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pcaTestButton">
             <property name="text">
              <string>PCA测试</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ldaTestButton">
             <property name="text">
              <string>LDA测试</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>0</height>
       </size>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>448</width>
         <height>502</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QTextEdit" name="outputEdit">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1247</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="trainAction"/>
    <addaction name="testAction"/>
    <addaction name="expressionAction"/>
    <addaction name="generateTrainAction"/>
    <addaction name="generateTestAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="trainAction">
   <property name="text">
    <string>指定训练目录</string>
   </property>
  </action>
  <action name="testAction">
   <property name="text">
    <string>指定测试目录</string>
   </property>
  </action>
  <action name="expressionAction">
   <property name="text">
    <string>指定表情库</string>
   </property>
  </action>
  <action name="generateTrainAction">
   <property name="text">
    <string>生成训练集</string>
   </property>
  </action>
  <action name="generateTestAction">
   <property name="text">
    <string>生成测试集</string>
   </property>
  </action>
  <action name="generateFaceAction">
   <property name="text">
    <string>生成表情库</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string> 关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
